diff --git a/dist/index.d.ts b/dist/index.d.ts
index 0d10e3993019d08a97151b9fec142fd3d51cd950..67d057e2ec16a5707d6137eeb1ce70fe2864be6c 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -574,4 +574,4 @@ declare const defineConfig: (options: Options | Options[] | ((
 overrideOptions: Options) => MaybePromise<Options | Options[]>)) => Options | Options[] | ((overrideOptions: Options) => MaybePromise<Options | Options[]>);
 declare function build(_options: Options): Promise<void>;
 
-export { type Format, type NormalizedOptions, type Options, build, defineConfig };
+export { type Format, type NormalizedOptions, type Options, type Plugin, build, defineConfig };
